### ISSUE TITLE
@prop({defaultValue})  bug

### ISSUE THREAD

**Issue by chobijaeyu** [rxweb/rxweb#205]
# Describe the bug
model:
```
export class User {
	@prop()
	id: string;
	@prop()
	shopowner?: any;
	@prop({defaultValue:[2]})
	groups:number[];
	@required()
	username: string;
	@required()
	password: string
}
```

component:
```
  ngOnInit() {
    let user = new User()
    this.UserForm = this.fb.formGroup(user)
}
```

html:
```
<form [formGroup]="UserForm">
 <input type="hidden" formControlName='groups' name='groups'>

 <input matInput formControlName='username' name='username' required>
 <input type="password" matInput formControlName='password' name='password'>
 <input matInput type="password" required>

</form>
```

UserForm.value:
```
{
  "id": null,
  "shopowner": null,
  "username": null,
  "password": null
}
```

the **groups**  just disappear，

change to 
```
	@prop({defaultValue:[]})
	groups:number[];
```

UserForm.value:
```
{
  "id": null,
  "shopowner": null,
  "groups": [],
  "username":null ,
  "password": null
}
```

# Expected Behavior
what I expect :
default set value like:
```
{
  "id": null,
  "shopowner": null,
  "groups": [2],
  "username":null ,
  "password": null
}
```

where am I wrong? or is that a bug?
thanks for your time.

# Package Version
"@rxweb/reactive-form-validators": "^1.9.2",

**Comment by ajayojha** [comment]
@chobijaeyu  I have created the stackblitz example as per your provided code and tried with the mentioned steps. It's working fine. Please refer this link (https://stackblitz.com/edit/angular-xjyuv8-ebqyzn?file=src/app/prop-default.component.html) and share your feedback if I am missing something.

**Comment by chobijaeyu** [comment]
@ajayojha 
[`here is my code`](https://stackblitz.com/edit/angular-b1hrqa)

the "groups" just disappear when you try input any thing .
how I wish it just work like you did.
plz tell me what I missing
![groups](https://user-images.githubusercontent.com/25117318/65104105-4f36d600-da0b-11e9-8205-c8a7d6fecb38.gif)

**Comment by ajayojha** [comment]
It's a bug :( and this will be resolved in our next release.
One Question, I am curious to know how you are using the array value of FormControl in your form. What's is the use-case of the Array value. Would be great if you share some information.

**Comment by chobijaeyu** [comment]
ok,
thanks for you and your brilliant project.

Array value question
In this case ,it's using for django permission group.


most using for one to many relational object.
[I change code for example](https://stackblitz.com/edit/angular-b1hrqa)

component:
```
    const order = new Order();
    order.SoldGoods = new Array<SoldGoods>();
    const soldGoods = new SoldGoods();
    order.SoldGoods.push(soldGoods);
    this.AddOrderForm = this.fb.formGroup(order);

  onMoreGoodsArray() {
    const soldGoodsArray = this.AddOrderForm.controls.SoldGoods as FormArray;
    soldGoodsArray.push(this.fb.formGroup(SoldGoods));
  }
```

models:
```
export class SoldGoods {
...
}
export class Order {
...
    @propArray(SoldGoods)
    SoldGoods: SoldGoods[];
}
```

html:
```
 <div class="form-row form-group" 
   [formGroup]="soldGoods"
   *ngFor="let soldGoods of AddOrderForm.controls['SoldGoods']['controls']; let i = index">

<input matInput formControlName='Sn' name='Sn' required>
<input type="number" matInput formControlName='SoldNum' name='SoldNum' required>
...

</div>

 <button mat-raised-button (click)="onMoreGoodsArray()">add more soldGoods</button>


```

**Comment by ajayojha** [comment]
@chobijaeyu Thank you very much for sharing the detailed information and provided the stackblitz example :). 

Will update you as soon as I release the new version, Maybe tomorrow.

**Comment by chobijaeyu** [comment]
hi @ajayojha 
I see you update code,
but there is bug when I reinstall package,
plz tell me what I'm missing?
package.json:
`    "@rxweb/reactive-form-validators": "^1.9.3",`

**Comment by ajayojha** [comment]
@chobijaeyu , I have fixed the issue in the repo but I haven't released yet. I was busy in so many other activities. Will release tomorrow. I will update you once I release the same.

**Comment by ajayojha** [comment]
@chobijaeyu I have released the new version 1.9.4, Please update the package in your project.

**Comment by chobijaeyu** [comment]
thank you
